simple-cipher/simple-cipher.js

```diff
@@ -0,0 +1,55 @@
+const alphabet = "abcdefghijklmnopqrstuvwxyz".split('');
+
+function generateNumber (min, max) {
+  var randNumber = Math.floor(Math.random() * (max - min + 1)) + min;
+  if (randNumber === max) {
+    randNumber = generateNumber(min, max);
+  }
+  return randNumber;
+}
+
+function numberToLetter (min, max) { 
+  return alphabet[generateNumber(min, max)];
+}
+
+function generateRandomKey (min, max, len) {
+  var key = '';
+  while (key.length < len + 1) {
+    key += numberToLetter(min, max);
+  }
+  return key;
+};
+
+function verifyBadKey (key) { 
+  return (!/^[a-z]+$/.test(key));
+};
+
+function _throw (msg) {throw new Error(msg)};
```

Comment:
What about the stuff we talked on telegram? :p

Reply:
I must have missed it :D I'm kinda surprised that I haven't corrected it before committing...